src/Microsoft.ML.TensorFlow/TensorflowCatalog.cs

```diff
@@ -31,6 +51,19 @@ public static class TensorflowCatalog
             string[] inputColumnNames)
             => new TensorFlowEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnNames, inputColumnNames, modelLocation);
 
+        /// <summary>
+        /// Scores a dataset using a pre-traiend TensorFlow model specified via <paramref name="tensorFlowModel"/>.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="tensorFlowModel">The pre-trained TensorFlow model.</param>
```

Comment:
>The pre-trained TensorFlow model. [](start = 42, length = 33)

I would call it pre-loaded instead of pre-trained. They all pre-trained I believe, but this one you load manually in memory, and for other you just pass path to model.